cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillController.java

```diff
@@ -449,37 +441,160 @@ public void onSuccess(V1Status v1Status, int statusCode, Map<String, List<String
       String name = meta.getName();
       appsApi.deleteNamespacedStatefulSetAsync(name, kubeNamespace, null, null, null, null, null, null,
                                                new ApiCallbackAdapter<V1Status>() {
+                                                 @Override
```

Comment:
Can you do the alignment like the old code?

Reply:
Ah yea, missed this one.